nservicebus/messaging/transactionalsession.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Transactional Session
```

Comment:
I think this one can be deleted altogether? We have the documentation under "nservicebus/transactional-session"